### PR TITLE
Update UserGuide to include contact find and contact edit

### LINKED ISSUE #170: Update User Guide to update contact commands
Update User Guide to update contact find and contact edit commands

### PR DESCRIPTION
**Pull request by Chronoxy**
Fixes #170

### CODE REVIEW

**Review by Ma-Xizi — APPROVED** [review]

**Comment by Ma-Xizi** [comment]
LGTM!

**Review by NorbertLoh — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -140,18 +140,19 @@ Format: `edit INDEX [/nNAME] [/pPHONE] [/eEMAIL] [/aADDRESS] [/tTAG]…​`
 * When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
 * You can remove all the person’s tags by typing `/t` without
   specifying any tags after it.
+* Role is not allowed to be edited!
```
Comment (ethantyh692) [inline docs/UserGuide.md:143]: Small nit. Grammar seems abit off here. Perhaps could say "Role is not editable" or "Role cannot be edited".

**Review by ethantyh692 — COMMENTED** [review]
Small nits to fix and consider

File: docs/UserGuide.md (hunk 1 above)
Comment (Chronoxy) [reply docs/UserGuide.md]: fixed

**Review by NorbertLoh — APPROVED** [review]
LGTM!

**Pull request merged**